authLdap.php

```diff
@@ -350,9 +349,8 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
         if ($uid) {
             // found user in the database
             authLdap_debug('The LDAP user has an entry in the WP-Database');
-            $user_info['ID'] = $uid;
-            unset ($user_info['display_name'], $user_info['nickname']);
-            $userid = wp_update_user($user_info);
+            $userid = $uid;
```

Comment:
We will need this ```wp_update_user``` as we want changes in the ldap (like f.i. email-address or name changes) to be passed to wordpress.